kolibri/content/migrations/0001_initial.py

```diff
@@ -42,8 +42,7 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='ContentKind',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('kind', models.CharField(blank=True, max_length=200, null=True)),
+                ('kind', models.CharField(choices=[(b'topic', 'Topic'), (b'video', 'Video'), (b'audio', 'Audio'), (b'exercise', 'Exercise'), (b'document', 'Document'), (b'image', 'Image')], max_length=200, primary_key=True, serialize=False)),
```

Comment:
This should be `choices=content_kinds.choices`. Otherwise you're just duplicating everything here, which defeats the purposes of having constants.